free-programming-books-id.md

```diff
@@ -73,7 +73,7 @@
 ### MySQL
 
 * [Tutorial MySQL untuk Pemula Hingga Mahir](https://umardanny.com/tutorial-mysql-untuk-pemula-hingga-mahir-ebook-download-pdf/)
-* [3 Days With Mysql For Your Application: Mysql Untuk Pemula](https://play.google.com/store/books/details/Onesinus_Saut_Parulian_3_Days_With_Mysql_For_Your?id=MbdTDwAAQBAJ) - Onesinus Saut Parulian, Google Play Books
+* [Mysql For Your Application: Mysql Untuk Pemula](https://play.google.com/store/books/details/Onesinus_Saut_Parulian_3_Days_With_Mysql_For_Your?id=MbdTDwAAQBAJ) - Onesinus Saut Parulian, Google Play Books
 
 ### Node.js
```

Comment:
Hello

- "Mysql.." is after "Tutorial" but it should be before because M is before T in the [English Alphabet](https://en.wikipedia.org/wiki/English_alphabet).
- Also, there should be 2 line space after the last resource (in this case Tutorial MySQL) in MySQL section

Please read [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) for more clarifications.